jfsd/main.py

```diff
@@ -1029,13 +1027,9 @@ def update_positions(
 
             # check that thermal fluctuation calculation went well
             if ((not math.isfinite(stepnormff)) or ((n_iter_Lanczos_ff > 150) and (stepnormff >  0.003))):
-                overlaps = utils.check_overlap(
-                    displacements_vector_matrix, 2.)
-                if(overlaps > 0):
-                    print(
-                        'Warning! Particles are overlapping. Reducing the timestep might help prevent the simulation from stopping.')
-                    raise ValueError(
-                        f"Far-field Lanczos did not converge! Stepnorm is {stepnormff}, iterations are {n_iter_Lanczos_ff}. Overlapping pairs are {overlaps}. Eigenvalues of tridiagonal matrix are {diag_ff}. Abort!")
+                overlap_monitor(displacements_vector_matrix,N)
```

Comment:
```suggestion
                overlap_monitor(displacements_vector_matrix, N)
```
And also for all the other occurrences.

Reply:
Modified.